py/alphazero/dashboard/Graph/Graph.py

```diff
@@ -0,0 +1,95 @@
+import networkx as nx
+from networkx.drawing.nx_agraph import graphviz_layout
+import matplotlib.pyplot as plt
+
+class Graph:
```

Comment:
`Graph` feels like a bad name for this class. Perhaps `SearchViz` instead?

Whatever you rename it to, I would rename the file to the same name.